REPOSITORY-GUIDELINES.md

```diff
@@ -0,0 +1,96 @@
+
+Examples of repositories of this type include useful Go libraries like [Fake](https://github.com/cilium/fake), or repositories designed to integrate Cilium with other projects like [Proxy](https://github.com/cilium/proxy).
+
+You can find the list of all ```Ecosystem``` repositories in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md#ecosystem-scope).
+
+### ```Infra```
+
+Infrastructure repositories are those that are required to keep the Cilium project running smoothly, and are not designed to be used by adopters.
+
+Examples of repositories of this type include development and testing tools like [little-vm-helper](https://github.com/cilium/little-vm-helper), or those helping with the overall contribution process like [github-actions](https://github.com/cilium/github-actions).
+
+You can find the list of all ```Infra``` repositories in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md#infra-scope).
+
+### ```Special```
+
+Special repositories are non-archived repositories that do not fit neatly into ```Core```, ```Ecosystem```, or ```Infra```. They are generally not designed to be used by adoptors.
+
+They include the [Community](https://github.com/cilium/community) and [.github](https://github.com/cilium/.github) repository, template repositories like [design-cfps](https://github.com/cilium/design-cfps), as well as all forks.
+
+You can find the list of all ```Special``` repositories in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md#special-scope).
+
+### ```Historical```
+
+Historical repositories are considered deprecated, are no longer maintained, and are converted to an archived, read-only state. They are not recommended for use by adoptors.
+
+Repositories that have been inactive for quite some time may be good candidates for archival. They can be proposed through the same process as other [changes of scope](#change-of-scope).
+
+You can find the list of all ```Historical``` repositories in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md#historical-scope).
+
+## Lifecycle
+
+### Voting
+
+All changes that occur within the lifecycle of a repository have the same process:
+
+- An issue is opened via the appropriate issue template ([New Repository Template]() or [Change of Scope Template]())
+- Once the issue is opened, ample opportunity for discussion among both committers and the larger Cilium community is provided (for example, 1 week)
+- If there are no objections among committers during this time and any relevant criteria are met (for example, the criteria found in [Addition](#addition)), the change is approved
+- If there are any objections from committers, the committers decide if the change should be approved via a simple majority vote
```

Comment:
We could reduce the voting to ```Core``` repositories only (both adding a new ```Core``` repository or changing an existing repository's scope to ```Core```). 

Part of this document was to address #82 so I think that some sort of voting is still necessary, although we could greatly reduce how often it happens this way.

Reply:
Limiting to `Core` sounds more pragmatic, yeah. Beyond that, I'd also suggest that this document describe that the voting process should adhere to the standard process in the Cilium project (with link to that), just so that it's clear that all the rules are the same, for instance the way that voting blocks apply.

Reply:
I have added some clarification, limiting formal voting to ```Core``` activities and linking to the appropriate voting section in our governance docs

Reply:
I think the intention here was never for it to actually come to a vote. As long as two committers think adding a repo is a good idea and no one objects then it is approved.

Reply:
:+1: I think the text could do with a little bit of wordsmithing on this front, I'm not sure it quite captures that sentiment as of now.

Generally speaking, this runs back to the way that we do decision-making in the project overall, and even this PR would be subject to a similar process - we'll refer this PR to the committers and provide a timeline to raise discussion, then assuming there are a couple of approvals and no objections/comments, we'll merge this PR.

Reply:
Ok, I have modified it a bit to try to better communicate our intentions. I have taken off the limit to Core repositories.